cypress/support/worldServices.js

```diff
@@ -0,0 +1,5 @@
+export default {
+  igbo: { url: '/igbo', article: undefined },
```

Comment:
Can this not combine with `cypress/support/config.js` with a small refactor?

Reply:
Agreed, this is a todo. We need to sit down as a test team and figure out how best to structure this. I want to avoid putting data in config though as it could soon bloat to a level where it becomes hard to maintain.